packages/ui/src/council/modals/RecoverVoteStake/index.ts

```diff
@@ -0,0 +1,13 @@
+import BN from 'bn.js'
+
+import { ModalWithDataCall } from '@/common/providers/modal/types'
+import { Address } from '@/common/types'
+
+export * from './RecoverVoteStakeModal'
+export type RecoverVoteStakeModalCall = ModalWithDataCall<
+  'RecoverVoteStake',
+  {
+    address: Address
+    stake: BN
```

Comment:
AFAICS `stake` is not used in the final modal. If so, please remove it from type.

Reply:
`stake` is displayed in the transaction modal as the amount the user is freeing, in the footer and in the lock info.